### ISSUE TITLE
Guests not spawning after refusing until beds

### ISSUE THREAD

**Issue by Jag062421** [OrionFive/Hospitality#549]
**Describe the bug**
I originally clicked the option to refuse guests until I had guest beds back on day two. I have now placed guest beds, but haven't received any guests for over a game week. I have the option to allow guests turned on in the mod menu. The Disable Guests option is turned off.

Also, I'm using Fluffy's Mod Manager mod and it's telling me there is a updated version of Hospitality from my current 1.2.10 to 1.2.11. I'm subscribed on Steam with auto update and I don't see a 1.2.11 release here on Github. Where do I find 1.2.11?

**Console log**
https://gist.github.com/7cd72a9ac21c2b83ef3b957e09f31321

**Error stacktrace**
Not receiving error message.

If you think it helps, add screenshots to help explain the problem.

Fixing bugs takes a lot of time. Please consider [becoming a patron](https://www.patreon.com/orionmods) to support development.

**Comment by OrionFive** [comment]
Don't worry about 1.2.11. It's not released yet. But some of your other mods may be outdated. I can't tell from your log what the problem is. Maybe you didn't play long enough for guests to arrive. Please use the dev menu to trigger a guest event (top left somewhere in the action menu). If they don't show up, make another log. It should now contain the needed information.

**Comment by Jag062421** [comment]
Thanks for the quick reply. I'm receiving visitors again one game date after I tried a few things. I moved my guest beds, made a few new ones, turned the Allow Visitor option off then on between saves, and removed one mod that I never use; "Use that sniper's scope!" (https://steamcommunity.com/sharedfiles/filedetails/?id=2036363660). I'm not sure if any of this helped or I just need to wait one more day, but it's working now.
